1.0.2-draft/README.md

```diff
@@ -0,0 +1,91 @@
+# vector-tile-spec
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in
+this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## 1. Purpose
+
+This specification attempts to create a standard for encoding of tiled geospatial data that can be shared across clients.
+
+## 2. File format
+
+The vector tile encoding scheme encodes vector data for a tile in a space efficient manner. It is designed to be used in browsers or serverside applications for fast rendering or lookups of feature data.
+
+Vector Tiles use [Google Protocol buffers](https://developers.google.com/protocol-buffers/) as a container format. It is exclusively geared towards square pixel tiles in [Spherical Mercator projection](http://wiki.openstreetmap.org/wiki/Mercator).
+
+## 3. Internal structure
+
+A vector tile SHOULD consist of one or more named layers. 
+
+A layer MUST contain a field describing its name and a version of the vector tile spec from which it was created. A layer  MUST also contain one or more features. A layer MAY contain one or more sets of feature attributes.
+
+Features MAY contain an `id` and `tags` (attributes) and MUST contain either a `geometry` (either point, linestring, or polygon) or a `raster` field. A feature SHALL NOT contain both a `raster` and a `geometry` field. If a feature has a `geometry` field it MUST also have a `type` field to describe the `geometry`. 
+
+### 3.1. Geometry Encoding
+
+Geometries are stored as an a single array of integers that represent a command,x,y stream (where command is a rendering command like `move_to` or `line_to`). Commands are encoded only when they change.
+
+Geometries with multiple parts (multipoint, multiline, or multipolygon) MUST be encoded one after another in the same `geometry` field and therefore are "flattened". Geometries with only a single part MUST have only a single `move_to` present. For multipoints and multilines a repeated `move_to` SHALL indicate another part of a multipart geometry. For polygons a repeated `move_to` SHALL indicate either another exterior of a new polygon part or an interior ring of the previous polygon part. The winding order MUST be used to distinguish between the two types: polygon interior rings (holes) must be oriented with the opposite winding order than their parent exterior rings and all interior rings MUST directly follow the exterior ring to which they belong. 
```

Comment:
Should the last in a ring point be the same as the first point, or are polygon rings implicitly closed? Section 6.1.7.1 of the ["Simple" Features spec](http://www.opengeospatial.org/standards/sfa) says that linear rings must be both simple and closed explicitly (first point = last point), but I don't think that gains us anything. Instead, could this spec say something like "exterior and interior rings MAY be closed. If they are not, then client software SHOULD treat them as implicitly closed (e.g: by appending the first point)."